.gitattributes

```diff
@@ -1,4 +1,17 @@
-# git config merge.ignore.name 'ignore changes merge driver'
-# git config merge.ignore.driver 'touch %A'
-lib/chef/version.rb merge=ignore
-*.reg text eol=crlf
```

Comment:
these probably should not get dropped.  i vaguely think i added the latter one, not sure what the former one does but it looks like it is likely tied to expeditor.

Reply:
corrected just now by BTM